  <launch>
    <arg name="waitTimeSecond" default="5"/>

    <node name="exp2_image_subscriber" pkg="adr_ros_tutorial" type="exp2_image_subscriber" output="screen">
      <param name="waitTimeSecond" value="$(arg waitTimeSecond)"/>
      <remap from="image_topic" to="/uav/camera/left/image_rect_color"/>
    </node>
  </launch>